packages.yml

```diff
@@ -1,3 +1,4 @@
-packages:
-  - package: fivetran/netsuite_source
-    version: [">=0.9.0", "<0.10.0"]
+packages:  
```

Comment:
@jmongerlyra I used this to test your source branch, feel free to modify this back around, didn't mean to push this to your branch.